repo/hints/hints.go

```diff
@@ -48,14 +51,16 @@ var (
 		CompressionNone:   compress.AlgoUnknown,
 		CompressionLZ4:    compress.AlgoLZ4,
 		CompressionSnappy: compress.AlgoSnappy,
+		CompressionZstd:   compress.AlgoZstd,
 		CompressionGuess:  compress.AlgoUnknown,
 	}
 
 	compressionSortMap = map[CompressionHint]int{
 		CompressionNone:   0,
 		CompressionLZ4:    1,
 		CompressionSnappy: 2,
-		CompressionGuess:  3,
+		CompressionZstd:   3,
+		CompressionGuess:  4,
```

Comment:
I would move ComressionGuess to `1`. It is likely we will add/remove algorithms, moving guess number
will be inconvinient

Reply:
It's purely for sorting and that's mostly for visuals, but sure I can change it.